### DISCUSSION TITLE
What's the last version that runs with python 3.8 ?

### DISCUSSION THREAD

**Discussion by AmjadHD** [gitpython-developers/GitPython#1468] [Q&A]
A quick glance at the source code I see usage of the pipe operator (`T1 | T2`) which isn't supported in 3.8.

**Comment by florisla** [comment]
It seems the linters are being run with 3.7, 3.8, 3.9, 3.10.

See https://github.com/gitpython-developers/GitPython/blob/main/.github/workflows/pythonpackage.yml .
